src/components/Main.js

```diff
@@ -47,7 +39,12 @@ class Main extends React.Component {
 
     return (
       <div className={classes.root}>
-        <MetaDiagram metaNodes={metaModel.mechanisms} metaLinks={metaModel.projections} componentsMap={componentsMap} />
+        <MetaDiagram metaNodes={metaModel.mechanisms} metaLinks={metaModel.projections} componentsMap={componentsMap}
+          metaTheme={{
+            customThemeVariables: {},
```

Comment:
@vidhya-metacell can't we just use a PsyNeuLinkView (MUI or MetaDiagram) theme?

Reply:
@zsinnema yes we are using meta-diagram theme, customThemeVariables gives us an option to make changes to variables for colors, fonts etc according to the project need.